components/donateform.js

```diff
@@ -93,20 +93,30 @@ class DonateForm extends Component {
     if (redirectSuccess) return (<div />)
 
     return (
-      <form className='flex flex-column' onSubmit={this.donate}>
+      <form className='flex flex-column f4-m' onSubmit={this.donate}>
         <div className='error tf-lato tc'>
           <p className='red'>{this.state.error}</p>
         </div>
+
+        {/* <TwoItemSwitcher
```

Comment:
Is this something you will be adding to and uncommenting in the future? Or can this be removed?

Reply:
Oops, missed that one. It's removed now.